### PR TITLE
Add EventInstance scaffold

### LINKED ISSUE #61: Event Instance Model
- Event 
- Dates
  - start and end date
- Location
   - free text
- URL

### PR DESCRIPTION
**Pull request by hackmastera**
closes #61

### CODE REVIEW

**Comment by kevinreiss** [comment]
👍

File: app/models/event_instance.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class EventInstance < ApplicationRecord
+  belongs_to :event
```
Comment (carolyncole) [inline app/models/event_instance.rb]: I wonder if we should put the analogous has_many in the event.

Comment (hackmastera) [reply app/models/event_instance.rb]: I will add it; might as well go ahead and do it in this PR!

**Comment by hackmastera** [comment]
@carolyncole I added the `has_many`; how does it look?

**Pull request merged**